docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -253,484 +133,518 @@ For example, if you ran the ``easylink run`` command on your laptop, the impleme
+  0            0_4          0_8          95  584-16-0130  ...                   00000       W2     2020         0
+  1            0_5          0_8          29  854-13-6295  ...                   00000       W2     2020         1
+  2            0_5          0_8          30  854-13-6295  ...                   00000       W2     2020         2
+  3         0_5621       0_2289          46  674-27-1745  ...                   00000       W2     2020         3
+  4         0_5623       0_2289          83  794-23-1522  ...                   00000       W2     2020         4
+  ...          ...          ...         ...          ...  ...                     ...      ...      ...       ...
+  9898     0_18936       0_7621          23  006-92-7857  ...                   00000       W2     2020      9898
+  9899     0_18936       0_7621          90  006-92-7857  ...                   00000       W2     2020      9899
+  9900     0_18937       0_7621           1  182-82-5017  ...                   00000     1099     2020      9900
+  9901     0_18937       0_7621         105  182-82-5017  ...                   00000     1099     2020      9901
+  9902     0_18939       0_7621           9  283-97-5940  ...                   00000       W2     2020      9902
+  [9903 rows x 25 columns]
+
+  >>> pd.read_parquet("known_clusters.parquet")
+  Empty DataFrame
+  Columns: [Input Record Dataset, Input Record ID, Cluster ID]
+  Index: []
 
-   $ pqprint results/2025_04_02_09_37_40/result.parquet
-         foo bar  counter  ...  added_column_1713  added_column_1714  added_column_1715
-   0         0   a     1715  ...               1713               1714               1715
-   1         1   b     1715  ...               1713               1714               1715
-   2         2   c     1715  ...               1713               1714               1715
-   3         3   d     1715  ...               1713               1714               1715
-   4         4   e     1715  ...               1713               1714               1715
-   ...     ...  ..      ...  ...                ...                ...                ...
-   59995  9995   a      912  ...               1713               1714               1715
-   59996  9996   b      912  ...               1713               1714               1715
-   59997  9997   c      912  ...               1713               1714               1715
-   59998  9998   d      912  ...               1713               1714               1715
-   59999  9999   e      912  ...               1713               1714               1715
-
-   [60000 rows x 8 columns]
-
-.. image:: DAG-e2e-pipeline.svg
-   :width: 500
-
-.. todo::
-   Explain spark in above diagram
-
-
-``e2e/pipeline_expanded.yaml``
-------------------------------
-A longer, more complex pipeline.
+It can also be useful to setup an alias to more easily preview parquet files. Add the following to your 
+``.bash_aliases`` or ``.bashrc`` file, and restart your terminal.
```

Comment:
```suggestion
It can also be useful to set up an alias to more easily preview parquet files.
Run the following line to do so.
(If you want this alias to persist across terminal restarts, you can add it to your ``.bashrc`` or ``.bash_aliases`` in your home directory.)
```